cli/create_test.go

```diff
@@ -416,6 +416,60 @@ func TestCreateWithRichParameters(t *testing.T) {
 		assert.ErrorContains(t, err, "parameter \""+wrongFirstParameterName+"\" is not present in the template")
 		assert.ErrorContains(t, err, "Did you mean: "+firstParameterName)
 	})
+
+	t.Run("CopyParameters", func(t *testing.T) {
+		t.Parallel()
+
+		client := coderdtest.New(t, &coderdtest.Options{IncludeProvisionerDaemon: true})
+		owner := coderdtest.CreateFirstUser(t, client)
+		member, _ := coderdtest.CreateAnotherUser(t, client, owner.OrganizationID)
+		version := coderdtest.CreateTemplateVersion(t, client, owner.OrganizationID, echoResponses)
```

Comment:
I'd like to see another test where the workspace is on an older template version, and the "cloned" workspace is on the exact same version.

Reply:
A bit of surgery, and I think I got it.